### PR TITLE
[CI SKIP]Sync for 1.6 Release

### PR DESCRIPTION
**Pull request by sid-github**
Copied from/based on https://github.com/google/EarlGrey/pull/354. PTAL but I will be updating this with more latest changes.

### CODE REVIEW

File: EarlGrey/Action/GREYPinchAction.h [hunk 1]
```diff
@@ -39,12 +39,28 @@ typedef NS_ENUM(NSInteger, GREYPinchErrorCode) {
...
+ *  @param direction The direction of the pinch.
+ *  @param duration  The time interval for which the pinch takes place.
+ *  @param angle     Angle of the vector in radians to which the pinch direction is pointing.
+ *
+ *  @returns An instance of @c GREYPinchAction, initialized with a provided direction and
+ *           duration and angle.
+ */
+- (instancetype)initWithDirection:(GREYPinchDirection)direction
+                         duration:(CFTimeInterval)duration
+                    andPinchAngle:(double)pinchAngle;
```
Comment (khandpur) [inline EarlGrey/Action/GREYPinchAction.h]: andPinchAngle => pinchAngle ?

File: EarlGrey/Action/GREYPinchAction.m [hunk 2]
```diff
@@ -52,12 +52,25 @@ @implementation GREYPinchAction {
...
+   */
+  double _pinchAngle;
 }
 
 - (instancetype)initWithDirection:(GREYPinchDirection)pinchDirection
                          duration:(CFTimeInterval)duration {
-  NSString *name = [NSString stringWithFormat:@"Pinch %@ for duration %g",
-                        NSStringFromPinchDirection(pinchDirection), duration];
+  return [self initWithDirection:pinchDirection duration:duration
+                   andPinchAngle:kDefaultPinchAngle];
```
Comment (khandpur) [inline EarlGrey/Action/GREYPinchAction.m]: line break after `self initWithDirection:pinchDirection`

Comment (sid-github) [inline EarlGrey/Action/GREYPinchAction.m]: Done

**Comment by sid-github** [comment]
Example pods SWIFT is failing due to https://github.com/CocoaPods/CocoaPods/issues/6214 everything else LG.

**Pull request merged**